src/app/components/list/overview/index.html

```diff
@@ -75,6 +75,180 @@
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="label"
+      isOptional="true"
+    >
+    Specifies a label for the filter's summary item. The label defaults to the <sky-code>value</sky-code> of the filter. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="name"
+      isOptional="true"
+    >
+    Specifies the name of the filter. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="value"
+      isOptional="true"
+    >
+    Specifies the current value of the filter. This property accepts <sky-code>any</sky-code> values.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="ListSearchModel properties">
+    <sky-demo-page-property
+      propertyName="fieldSelectors"
+      isOptional="true"
+    >
+    Specifies the columns to search. The columns correspond to <sky-code>field</sky-code> values that you specify with the <sky-code>data</sky-code> property. This property accepts an array of <sky-code>string</sky-code> values. 
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="functions"
+      isOptional="true"
+    >
+    Specifies an array of functions that returns a <sky-code>boolean</sky-code> value of <sky-code>true</sky-code> when the search is successful. Acceptable values: <sky-code>(data: any, searchText: string) => boolean></sky-code>.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="searchText"
+      isOptional="true"
+    >
+    Specifies the text to search. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="ListSortFieldSelectorModel properties">
+    <sky-alert alertType="warning">The following properties are a work in progress and not recommended for use. </sky-alert>
+    <sky-demo-page-property
+      propertyName="descending"
+      isOptional="true"
+      defaultValue="false"
+    >
+    Indicates whether to sort in descending order. This property accepts <sky-code>boolean</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="fieldSelector"
+      isOptional="true"
+    >
+    Specifies the fields to sort. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="ListSortLabelModel properties">
+    <sky-alert alertType="warning">The following properties are a work in progress and not recommended for use.</sky-alert>
+    <sky-demo-page-property
+      propertyName="fieldType"
+      isOptional="true"
+    >
+    Specifies the label type. This property accepts <sky-code>string</sky-code>values.
```

Comment:
We're missing a space between the code snippet and "values."

Also, it might not matter since we recommend against using these properties, but I don't know what "label type" is. That doesn't really mean anything to me.

Reply:
Assuming that we decided explaining label type doesn't matter since we recommend against using this property...